### PR TITLE
Move pullspecs to bundle.env file

### PR DESCRIPTION
**Pull request by andreasgerstmayr**

### CODE REVIEW

File: Dockerfile.bundle [hunk 1]
```diff
@@ -8,9 +8,6 @@ RUN dnf install -y python3 python3-ruamel-yaml-clib python3-ruamel-yaml patch
 RUN mkdir /licenses
 COPY tempo-operator/LICENSE /licenses/.
 
-# Use a new stage to enable caching of the package installations for local development
-FROM builder-deps as builder
```
Comment (pavolloffay) [inline Dockerfile.bundle:12]: why this change?

File: bundle-patch/bundle.env [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+TEMPO_JAEGER_QUERY_IMAGE_PULLSPEC=quay.io/redhat-user-workloads/rhosdt-tenant/tempo/tempo-jaeger-query@sha256:ebef9709e328cf9918ff99ed3dcac6abb1a38c2cd3b46af4fd61cc0b87c0e165
+
+TEMPO_GATEWAY_IMAGE_PULLSPEC=quay.io/redhat-user-workloads/rhosdt-tenant/tempo/tempo-gateway@sha256:d129101bf8563715cf8f2776a8359316d8dde35899af7f736fe9cbb380f4530c
+
+TEMPO_OPA_IMAGE_PULLSPEC=quay.io/redhat-user-workloads/rhosdt-tenant/tempo/tempo-opa@sha256:6f91ab07ee9b0361fd4f26d0d380c09a43a1839099a97103e6c130b6cf926be8
+
+TEMPO_OPERATOR_IMAGE_PULLSPEC=quay.io/redhat-user-workloads/rhosdt-tenant/tempo/tempo-operator@sha256:89110559c33c815b59aa914b5dbb170242dbbdbf9cf1e9f75f63875fb6d9e895
+
+# TODO, we used to set the proxy image per OCP version
```
Comment (pavolloffay) [inline bundle-patch/bundle.env:17]: we can remove this

File: bundle-patch/bundle.env [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# The pullspec should be image index, check if all architectures are there with: skopeo inspect --raw docker://$IMG | jq
+# The names must match the names of the Component CRs in Konflux.
+# Separate variables by empty lines due to merge conflicts
+
+TEMPO_TEMPO_IMAGE_PULLSPEC=quay.io/redhat-user-workloads/rhosdt-tenant/tempo/tempo@sha256:7ad3f3e5f32457a2f2bf79e0ceffb9f988183a8fb4654d39f5d6496ca0ae9b70
```
Comment (pavolloffay) [inline bundle-patch/bundle.env:5]: Could you please document the convention to match with the component name?

**Review by pavolloffay — COMMENTED** [review]
I would be great to add a script which takes a snapshot and updates pullspecs

File: Dockerfile.bundle (hunk 1 above)
Comment (andreasgerstmayr) [reply Dockerfile.bundle:12]: Container layers are already cached unless a modified context is copied to a layer with `COPY`, but in this case the `COPY` is after the package installation, i.e. the installation layers are always cached.

Do you know why it was there in the first place?

File: bundle-patch/bundle.env (hunk 3 above)
Comment (andreasgerstmayr) [reply bundle-patch/bundle.env:5]: done

**Comment by andreasgerstmayr** [comment]
> I would be great to add a script which takes a snapshot and updates pullspecs

Done in `hack/update-bundle-env`

**Review by pavolloffay — APPROVED** [review]
